drivers/dac/ad5770r/ad5770r.c

```diff
@@ -420,16 +420,25 @@ int32_t ad5770r_set_dac_value(struct ad5770r_dev *dev,
 			      uint16_t dac_value, enum ad5770r_channels channel)
 {
 	uint8_t data[2];
+	int32_t ret;
 
 	if (!dev)
 		return FAILURE;
 
 	data[1] = (uint8_t)AD5770R_CH_DAC_DATA_LSB(dac_value);
 	data[0] = (uint8_t)((dac_value & 0x3FC0) >> 6);
 
-	return ad5770r_spi_reg_write_multiple(dev,
-					      AD5770R_CH0_DAC_MSB + 2 * channel,
-					      data, sizeof(data));
+	ret = ad5770r_spi_reg_write_multiple(dev,
```

Comment:
i think two commits got squashed here. This commit does more that what it's described in the commit message.

Reply:
indeed - I'll split back to two separate commits as it was before.